beats/filebeat/install.sls

```diff
@@ -5,5 +5,6 @@ include:
 filebeat_install:
     pkg.installed:
         - name: filebeat
+        - version: {{ salt['pillar.get']('beats:filebeat:version', '6.1.2') }}
```

Comment:
Why not something like that?

```yaml
[...]
{% if beats.filebeat.version is defined %}
- version: {{ salt['pillar.get']('beats:filebeat:version') }}
{% endif %}
[...]
```
If specified within the pillar, the version will be enforced. If not, the latest version of the package will be installed